rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -1119,40 +1124,49 @@ def multicall_specific(
             return output
         return [contract.decode(x, method_name, arguments[0]) for x in output]
 
-    def get_erc20_contract_info(self, address: ChecksumEvmAddress) -> dict[str, Any]:
-        """
-        Query an erc20 contract address and return basic information as:
-        - Decimals
-        - name
-        - symbol
-        At all times, the dictionary returned contains the keys; decimals, name & symbol.
-        Although the values might be None.
-        if it is provided in the contract. This method may raise:
-        - BadFunctionCallOutput: If there is an error calling a bad address
-        """
-        cache = self.contract_info_erc20_cache.get(address)
-        if cache is not None:
-            return cache
+    def get_multiple_erc20_contract_info(self, addresses: list[ChecksumEvmAddress]) -> dict[str, Any]:  # noqa: E501
+        all_info: dict[str, Any] = {}
 
-        properties = ('decimals', 'symbol', 'name')
-        info: dict[str, Any] = {}
+        if len(addresses) == 0:
+            return {}
```

Comment:
this can be executed before defining all_info

Reply:
Removed `all_info` since the function does not need to return anything